website/docs/r/site_recovery.html.markdown

```diff
@@ -0,0 +1,50 @@
+---
+layout: "vmc"
+
+page_title: "VMC: vmc_site_recovery"
+sidebar_current: "docs-vmc_site_recovery"
+
+description: |-
+  Provides a resource to activate and deactivate site recovery for an SDDC.
+---
+
+# vmc_site_recovery
+
+Provides a resource to activate and deactivate site recovery for an SDDC.
+~> **Note:** Site recovery resource implicitly depends on SSDC resource creation. SDDC must be provisioned before a site recovery can be activated. For details on how to provision an SDDC refer to [vmc_sddc](https://www.terraform.io/docs/providers/vmc/r/sddc.html).
```

Comment:
while you are at it, can you also add the SDDC doc link to public IP doc?
https://www.terraform.io/docs/providers/vmc/r/public_ip.html